Directory.Build.props

```diff
@@ -5,9 +5,8 @@
     <RepositoryUrl>https://github.com/mono/taglib-sharp</RepositoryUrl>
     <RepositoryType>git</RepositoryType>
 
-    <!-- Make sure to update Directory.Build.targets too! -->
-    <ExeTargetFrameworks>net6.0;net472</ExeTargetFrameworks>
-    <LibTargetFrameworks>net6.0;netstandard2.0</LibTargetFrameworks>
+    <ExeTargetFrameworks>net7.0;net472</ExeTargetFrameworks>
+    <LibTargetFrameworks>net7.0;netstandard2.0</LibTargetFrameworks>
```

Comment:
@xaviergxf Thanks for the PR. Could I have you leave the net6.0 and add net8.0 instead. net8.0 is an LTS version and was just barely released.

Reply:
I have updated it on my side but there are now problems regarding deprecation of Formatter-based serialization. I don't understand why this is used...

Reply:
oh fun :) ok, I'll try and take a peek at this too.